vip_hci/metrics/fakecomp.py

```diff
@@ -184,7 +187,10 @@ def cube_inject_companions(array, psf_template, angle_list, flevel, plsc,
                           '({:.2f} pxs)'.format(posx, posy, rad_arcs,
                                                 rad_dists[i]))
 
-    return array_out
+    if full_output:
+        return array_out, posx, posy
```

Comment:
This is only returning the x,y position of the last companion injected. The coordinates need to be arranged in a nested tuple perhaps.